CMakeLists.txt

```diff
@@ -6,8 +6,9 @@ set(PROJECT_NAME
 
 project(${PROJECT_NAME})
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g")
```

Comment:
This should happen automatically based on the cmake build type

Reply:
Removed